clusterpools/templates/install-config.gcp.yaml.template

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+metadata:
+  name: 
+baseDomain: 
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  replicas: 3
+  platform:
+    gcp:
+      type: n1-standard-4
+compute:
+- hyperthreading: Enabled
+  name: worker
+  replicas: 3
+  platform:
+    gcp:
+      type: n1-standard-4
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineCIDR: 10.0.0.0/16
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  gcp:
+    projectID: __CLUSTERPOOL_GCP_PROJECT_ID__
+    region: __CLUSTERPOOL_GCP_REGION__
+pullSecret: "" # skip, hive will inject based on it's secrets
```

Comment:
it's --> its

Reply:
@dhaiducek I don't know how to say it but I stole this install-config from the RHACM product's UI page so we have it wrong in the product!

Reply:
EVERYTHING'S BROKEN!!!